tpgtools/overrides/monitoring/beta/metrics_scope.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- type: UNDELETABLE
+- type: APPEND_TO_BASE_PATH
```

Comment:
```suggestion
# need to append version as dcl monitoring since the mm basepath declaration we share
# does not include version https://git.io/Jz4Wn
- type: APPEND_TO_BASE_PATH 
```